### ISSUE TITLE
Render example error in Ember 1.13

### ISSUE THREAD

**Issue by benmurden** [poteto/ember-cli-flash#96]
The following example
```handlebars
{{#each flashMessages.queue as |flash|}}
  {{#flash-message flash=flash as |component flash|}}
    {{#if flash.template}}
      {{render flash.template flash.context}}
    {{else}}
      <h6>{{component.flashType}}</h6>
      <p>{{flash.message}}</p>
    {{/if}}
  {{/flash-message}}
{{/each}}
```
throws the following error.
```
Uncaught Error: Assertion Failed: The first argument of {{render}} must be quoted, e.g. {{render "sidebar"}}.
```
It would seem one cannot use render in this way. Another possibility would be to replace this line with a [component helper](http://emberjs.com/blog/2015/03/27/ember-1-11-0-released.html#toc_component-helper), available since 1.11.
```handlebars
...
    {{#if flash.componentName}}
      {{component flash.componentName content=flash.content}}
    {{else}}
...
```

**Comment by poteto** [comment]
Ah thanks, I haven't updated the readme! Would you like to submit a PR for it? :)

**Comment by benmurden** [comment]
Sure! I'll do that now.

**Comment by HeroicEric** [comment]
I think this can be closed as a result of #97

**Comment by benmurden** [comment]
Agreed.
